cmd/catalog-importer/cmd/sync.go

```diff
@@ -187,250 +187,233 @@ func (opt *SyncOptions) Run(ctx context.Context, logger kitlog.Logger, cfg *conf
 
 	// Create missing catalog types
 	OUT("\n↻ Creating catalog types that don't yet exist...")
-	for _, outputType := range cfg.Outputs() {
-		logger := kitlog.With(logger, "type_name", outputType.TypeName)
-
-		baseModel, enumModels := output.MarshalType(outputType)
-	createCatalogType:
-		for _, model := range append(enumModels, baseModel) {
```

Comment:
This just simplifies down into a single loop with the `.AllOutputTypes()` method